packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerService.ts

```diff
@@ -434,7 +434,12 @@ export class OpenId4VcIssuerService {
 
       credentialResponse = vcIssuer.createCredentialResponse({
         credential: credentialRequest.proof ? credentials.credentials[0] : undefined,
-        credentials: credentialRequest.proofs ? credentials.credentials : undefined,
+        credentials: credentialRequest.proofs
```

Comment:
issuerMetadata is retrieved from `getIssuer`, which only takes the issuer and the context, so it won't really have the context of the specific issuance session i think?

for verification we store the version on the record. Which we might need to do here as well if we can't infer it based on the offer or request

Reply:
Or we can just decide to use the latest format, since we don't support deferred issuance for Draft < 15

Reply:
what about non-deferred?

Reply:
Done